.gitignore

```diff
@@ -152,6 +152,9 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+# Agent-generated log files
+logs/
```

Comment:
Will you ensure this is also ignored in the .dockerignore so we dont accidentally build any sensitive logs into the image

Reply:
ah thanks - we didn't have a .dockerignore file yet so I added one